diff --git a/backend/index.js b/backend/index.js
index 9f5540f..0ff3e49 100644
--- a/backend/index.js
+++ b/backend/index.js
@@ -4,7 +4,7 @@ const MAX_POSSIBLE_DANCE_POINTS_DIFFERENCE = 100;
 
 // CHANGE THESE TO MATCH THE WANTED GOOGLE SHEET
 // ---------------------------------------------
-const spreadsheetId = "1fa88lJihFOGGjR3yZiPHo70LOPRWlRLrFFn_I1yFJg4";
+const spreadsheetId = "12LaA-7rPKRIK2CLxfuzRkeuTl3KvGeaYB8fnaES2Qho";
 const scoresTabName = "Scores";
 // ---------------------------------------------
 
@@ -16,6 +16,16 @@ const dgram = require("dgram");
 const sanitize = require("sanitize-filename");
 const { google } = require("googleapis");
 
+//const myClient = new WebSocket("wss://eurocup2024tournamentmanager.azurewebsites.net/ws")
+const myClient = new WebSocket("ws://localhost:5232/ws")
+
+myClient.on('message', function message(data) {
+  console.log('received: %s', data);
+});
+
+myClient.on('open', function open() {
+});
+
 let scoreSendingQueue = [];
 
 console.log("Authenticating for google sheets...");
@@ -231,6 +241,8 @@ const processMessage = async (address, msg, isFinalScore, isFinalMarathonScore)
     const filePath = path.join(".", "scores", filename);
     fs.writeFileSync(filePath, json, "utf8");
 
+    myClient.send(json);
+
     // Store score in queue
     storeScoreForSending(scoreData);
   }
diff --git a/prod.bat b/prod.bat
index 667c9c0..587cdf7 100644
--- a/prod.bat
+++ b/prod.bat
@@ -1,4 +1,4 @@
 (
-   start "task1" cmd /C "cd frontend & npm run build-and-serve"
+   start "task1" cmd /C "cd frontend & npm start"
    start "task2" cmd /C "cd backend & npm run start"
 ) | pause
